src/bindings/python/src/pyopenvino/core/infer_request.cpp

```diff
@@ -636,6 +647,17 @@ void regclass_InferRequest(py::module m) {
             :rtype: List[openvino.runtime.ConstOutput]
         )");
 
+    cls.def(
+        "get_model_inputs",
```

Comment:
Why not `get_inputs()`? To be align with C++

Reply:
Discussed offline, please rereview.